iqa_common/executor/execution.py

```diff
@@ -127,18 +305,14 @@ def read_stderr(self, lines: bool=False):
         :type lines: bool
         :return: Stdout content as str if lines is False, or as a list
```

Comment:
Not part of this commit, but could you please update stdout -> stderr in docs. thanks :)